tools/revpi-set-mac

```diff
@@ -48,5 +53,5 @@ fi
 # write MAC Address
 for idx in {1..6}
 do
-    ethtool -E "${iface}" magic "${magic}" offset "${idx}" value 0x"${BASH_REMATCH[idx]}"
+    ethtool -E "${iface}" magic "${magic}" offset "${idx}" length 1 value 0x"${BASH_REMATCH[idx]}"
```

Comment:
Move this change to commit 51bbbec524e2 ("Fix issue with ethtool from Bullseye").

Reply:
Can't do that as `revpi-set-mac` was introduced in a later commit. I'll fix it in the initial commit of this file (1e9170b9f3bfb9e6e0bbeabaad24bafed03529dc)